addons/zookeeper/dataprotection/restore.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -o errexit
+set -o pipefail
+set -o nounset
+
+DATA_DIR=${DATA_DIR:-/bitnami/zookeeper/data}
+BACKUP_DIR=/data/backup
+
+echo "Restoring Zookeeper data directory..."
+
+LATEST_BACKUP=$(ls -t "${BACKUP_DIR}" | head -1)
```

Comment:
The backup dir may be empty, so the 'LATEST_BACKUP' is nil

Reply:
I just hard coded BACKUP_DIR=/data/backup. So what should it be?